Mathlib/Algebra/Group/Subgroup/ZPowers/Basic.lean

```diff
@@ -148,9 +148,12 @@ theorem Int.zmultiples_natAbs (a : ℤ) :
     AddSubgroup.zmultiples (a.natAbs : ℤ) = AddSubgroup.zmultiples a := by
   simp [le_antisymm_iff, Int.mem_zmultiples_iff, Int.dvd_natAbs, Int.natAbs_dvd]
 
-lemma AddSubgroup.closure_singleton_int_one_eq_top : closure ({1} : Set ℤ) = ⊤ := by
+lemma Int.addSubgroup_closure_one : AddSubgroup.closure ({1} : Set ℤ) = ⊤ := by
```

Comment:
We refer to namespaced names in camelCase:
```suggestion
lemma Int.addSubgroupClosure_one : AddSubgroup.closure ({1} : Set ℤ) = ⊤ := by
```
(this is a relatively new policy, so you'll see a lot of exceptions, but we shouldn't add any more).

Please update the PR title accordingly.